index.js

```diff
@@ -60,6 +60,8 @@ var defaults = function () {
       })
     },
     contents: require('./lib/contents'),
+    indentableSelector: ':first-child', // The element within the `.node` which will be used for showing tree indentations
```

Comment:
Is this for https://github.com/SpiderStrategies/Scoreboard/issues/15599?  If so, the name isn't making sense to me.  This would control what's initially expanded in the tree right?  So maybe something like `initiallyExpandedSelector` or `expandedSelector` would be more appropriate?

Reply:
No, this has nothing to do with https://github.com/SpiderStrategies/Scoreboard/issues/15599.

It's something I need for the gantt chart.

It also has nothing to do with "initial". Every node in a tree that is a descendant has a x transform. This allows someone to control which node receives that transformation.